### DISCUSSION TITLE
Compute IRR

### DISCUSSION THREAD

**Discussion by Domth13** [asreview/asreview#724] [Q&A]
What is the best way to calculate the Interrater-Reliability (IRR) out of two Results (Excel Sheets)?

**Comment by SagevdBrand** [comment]
Hi there @Domth13! Interesting question! Perhaps you could use Cohen's Kappa, see here on [this wikipedia page](https://en.wikipedia.org/wiki/Cohen%27s_kappa). Hopefully this will make things clear for you!
